src/routes/events.$eventId.tsx

```
@@ -1328,6 +1335,65 @@ clubs (name, slug, logo_url, primary_color, secondary_color),          event_met
+          if (distance <= targetRadius) {
+            try {
+              const res = await performAutoCheckIn(myRsvpId!);
+              if (res.status === "success" || res.status === "already_checked_in") {
+                toast.success("Welcome! You have been automatically checked in.");
+                refetch();
+              }
+            } catch (err) {
+              console.error("Auto check-in verification failed:", err);
+            }
+          } else {
+            toast.error(`You are too far from the venue to check in (${Math.round(distance)}m away).`);
```

ESLint

[error] Replace ``You·are·too·far·from·the·venue·to·check·in·(${Math.round(distance)}m·away).`` with `⏎··············`You·are·too·far·from·the·venue·to·check·in·(${Math.round(distance)}m·away).`,⏎············`

(prettier/prettier)